src/DeckSurf.SDK.sln

```diff
@@ -1,11 +1,11 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31321.278
+# Visual Studio Version 17
```

Comment:
Can you reverse the changes here before I merge? The solution file should remain unchanged.